packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -291,12 +291,14 @@ Future<Null> _buildGradleProjectV2(String gradle, BuildInfo buildInfo, String ta
   }
   if (buildInfo.previewDart2) {
     command.add('-Ppreview-dart-2=true');
-  if (buildInfo.strongMode)
-    command.add('-Pstrong=true');
-  if (buildInfo.extraFrontEndOptions != null)
-    command.add('-Pextra-front-end-options=${buildInfo.extraFrontEndOptions}');
-  if (buildInfo.extraGenSnapshotOptions != null)
-    command.add('-Pextra-gen-snapshot-options=${buildInfo.extraGenSnapshotOptions}');
+    if (buildInfo.trackWidgetCreation)
```

Comment:
It looks like checking of 'buildInfo.strongMode' was removed by mistake, and 2 identical checks for 'buildInfo.trackWidgetCreation' were inserted.

Reply:
doh. bad merge. fixed.